todo/src/actions/index.js

```diff
@@ -0,0 +1,31 @@
+export const ADD_TODO = "ADD_TODO";
+export const addTodo = todo => {
+    return {
+        type: ADD_TODO, 
+        payload: todo
+    }
+}
+
+export const TOGGLE_COMPLETE = 'TOGGLE_COMPLETE';
+export const toggleComplete = id => {
+    return {
+        type: TOGGLE_COMPLETE,
+        payload: id
+    }
+}
+
+export const DELETE_TODO = 'DELETE_TODO';
```

Comment:
Good structure, clear concise code. I love how you are exporting the const variable next to the action creator. Good stuff